radixdlt-core/radixdlt/src/main/java/org/radix/GenerateUniverses.java

```diff
@@ -402,7 +402,10 @@ private static void outputUniverse(
 				universe.put("pubkey", k.getPublicKey().toBase64());
 				universe.put("address", universeAddress.toString());
 				universe.put("token", tokenRri.toString());
-				universe.put("value", Bytes.toBase64String(universeBytes));
+
+				JSONObject universe_json = new JSONObject(serialization.toJson(p.getSecond(), Output.WIRE));
```

Comment:
I don't think we use snake case (universe_json)

Reply:
Fixed